src/network.rs

```diff
@@ -2,6 +2,8 @@
 //!
 //! This module implements a safe wrapper around the networking functions found in ``network.h``.
 
+#![allow(clippy::bad_bit_mask)]
```

Comment:
What is this about?

Reply:
Clippy currently throws errors for &-bitmasking with zero in the `bitflags!` macro, which occur because some of the flags in the macro have a value of zero. It links [here](https://rust-lang.github.io/rust-clippy/master/index.html#bad_bit_mask) for more details.

<details><summary>Here's an example of the output:</summary>

```
error: &-masking with zero
  --> src/network.rs:50:1
   |
50 | / bitflags! {
51 | |     /// Incoming Address Routing
52 | |     pub struct AddressRouting: u32 {
53 | |         const INADDR_ANY        = 0;
54 | |         const INADDR_BROADCAST  = 0xffffffff;
55 | |     }
56 | | }
   | |_^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#bad_bit_mask
   = note: `#[deny(clippy::bad_bit_mask)]` on by default
   = note: this error originates in the macro `__impl_bitflags` which comes from the expansion of the macro `bitflags` (in Nightly builds, run with -Z macro-backtrace for more info)
```
</details>

Since clippy is throwing errors, preventing CI from passing, and I'm not sure if the `bitflags!` approach is appropriate, so I figured I'd have it be ignored for now until someone can take a look at it.

Reply:
Oh okay!